ci-operator/config/openshift/oadp-operator/openshift-oadp-operator-oadp-1.3__4.12.yaml

```diff
@@ -2,13 +2,11 @@ build_root:
   image_stream_tag:
     name: builder
     namespace: ocp
-    tag: rhel-8-golang-1.20-openshift-4.14
+    tag: rhel-8-golang-1.19-openshift-4.12
```

Comment:
I would undo these changes of image in 1.3 branch, it goes against our standards https://github.com/openshift/oadp-operator/blob/master/docs/developer/PROW_CI.md?plain=1#L73-L81

i.e., 4.12 and 4.13 jobs would use Go 1.19, but 4.14 and 4.15 jobs use Go 1.20

Reply:
why 4.12 and 4.13 should not use go 1.20 ? what about oadp 1.0 and branch 4.14?

I think having per ocp image makes sense as we are testing as well variety of go versions almost for free here.